Gemfile

```diff
@@ -88,12 +91,12 @@ group :development do
   gem "binding_of_caller"
   gem "better_errors"
   gem "bullet"
+  gem 'capistrano-delayed-job', '~> 1.0'
```

Comment:
We don't use Capistrano anymore so we don't need to include this :). I should get to removing Capistrano at some point!